sdks/python/apache_beam/runners/worker/sdk_worker_main.py

```diff
@@ -76,6 +77,8 @@ def create_harness(environment, dry_run=False):
   RuntimeValueProvider.set_runtime_options(pipeline_options_dict)
   sdk_pipeline_options = PipelineOptions.from_dictionary(pipeline_options_dict)
   filesystems.FileSystems.set_options(sdk_pipeline_options)
+  pickler.set_pipeline_options(
```

Comment:
i'd rename this as this sounds as if we were setting pipeline options. Alternatives to consider:
- call this method `pickler.set_options` (to mean that we set picker's options; in which case the corresponding pipeline option could be called `--pickle_options`, sample value `--pickle_options=use_dill`)
- `pickler.set_pickle_library(pickle_libary=sdk_pipeline_options.view_as(SetupOptions).pickle_library)` (my preference.)
- `pickler.configure_from_options(pickle_options=sdk_pipeline_options.view_as(SetupOptions).pickle_options))`

Reply:
I've renamed this to set_library, and I think we should only have a single set_library function that takes a string as a parameter.